kubernetes/configs/prod/commons.sh

```diff
@@ -39,7 +39,6 @@ SELFCARE_V2_URL="https://api.selfcare.pagopa.it/external/v2"
 SELFCARE_PRODUCT_ID="prod-interop"
 
 PRIVACY_NOTICES_DYNAMO_TABLE_NAME="interop-privacy-notices-prod"
-PRIVACY_NOTICES_ACCEPTANCE_DYNAMO_TABLE_NAME="interop-privacy-notices-acceptances-prod"
```

Comment:
same

Reply:
AFAIK @Carminepo2 isn't using this table in its job. Is it used by someone other?

Reply:
this table is used in the BFF, not in the job